BLSQ/iaso
# IA-2153: Make the difference between allowed-to-create children types and children-type to show in Iaso

Have a way on mobile to differentiate the org unit type we would like to allow people to suggest adding, and those that are fixed and are not in the scope of suggestions for additions (ie regions, districts)

Related JIRA tickets : IA-2153

## Self proofreading checklist


## Changes

- model and migration
- duplicate code for sub unit type creation
- adding custom tooltip

## How to test

Create or edit an org unit type.
You should be able to add sub unit types displayed and sub unit types allowed for the creation in the mobile app
I have added same check on circular hierarchy as the one sub unit types

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/2aa2dcec-fe79-4bcb-bb1d-28077d5acf59
